src/components/Panel/Rightside/Members.vue

```diff
@@ -91,31 +92,64 @@ export default {
 
   data () {
     return {
-      userQuery: null,
+      query: null,
+
+      members: [],
+      nonMembers: [],
     }
   },
 
   computed: {
     ...mapGetters({
       users: 'users/list',
     }),
+  },
 
-    members () {
-      return this.users.filter(u => this.channel.isMember(u.userID))
+  watch: {
+    'query' (query) {
+      if (query) {
+        const nonMembers = []
+        const limit = 15
```

Comment:
A bit overkill -- `userList(({ query, limit }))` already defines that 15 is limit, so no need to worry about magic-numbers